Source/VMware.vSphereDSC/Tests/Integration/VMHostVss.Integration.Tests.ps1

```diff
@@ -84,29 +84,15 @@ $script:mofFileWithRemoveVss = "$script:integrationTestsFolderPath\$($script:con
 try {
     Describe "$($script:dscResourceName)_Integration" {
         Context "When using configuration $($script:configWithNewVss)" {
-            BeforeEach {
-                # Arrange
-                $startDscConfigurationParameters = @{
-                    Path = $script:mofFileWithNewVss
-                    ComputerName = 'localhost'
-                    Wait = $true
-                    Force = $true
-                }
-
-                # Act
-                Start-DscConfiguration @startDscConfigurationParameters
-            }
-
             It 'Should compile and apply the MOF without throwing' {
+                # Assert
                 $startDscConfigurationParameters = @{
                     Path = $script:mofFileWithNewVss
                     ComputerName = 'localhost'
                     Wait = $true
                     Force = $true
                 }
-
-                # Assert
-                { Start-DscConfiguration @startDscConfigurationParameters } | Should -Not -Throw
+                Start-DscConfiguration @startDscConfigurationParameters | Should -Not -Throw
```

Comment:
Change it to:

```
 { Start-DscConfiguration @startDscConfigurationParameters } | Should -Not -Throw
```

And the same for all other tests of the same type.

Reply:
Changed